include/swift/AST/Attr.def

```diff
@@ -292,6 +292,10 @@ DECL_ATTR(_restatedObjCConformance, RestatedObjCConformance,
 SIMPLE_DECL_ATTR(_downgrade_exhaustivity_check, DowngradeExhaustivityCheck,
                  OnEnumElement | LongAttribute | UserInaccessible, 71)
 
+SIMPLE_DECL_ATTR(_implicitly_unwrapped_optional,
+                 ImplicitlyUnwrappedOptional,
+                 OnFunc | OnVar | RejectByParser, 72)
```

Comment:
Please bump the module format version number too, since modules with this attribute aren't compatible with the existing compiler.